### ISSUE TITLE
Clarify using underscores to denote private variables

### ISSUE THREAD

**Issue by ShatReal** [godotengine/godot-docs#4913]
**Your Godot version:**
3.3

**Issue description:**
The documentation states

> Prepend a single underscore (_) to virtual methods functions the user must override, private functions, and private variables

However, this doesn't mention anything about private Constants and Enums. What should one do for them? What about export variables? Can they be private?

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html

**Comment by pycbouh** [comment]
Yes, you can denote everything as private, consts, enums, inner classes.

**Comment by ShatReal** [comment]
Having an Enum called _State looks weird lol

**Comment by pycbouh** [comment]
You don't have to use this notation, it's not really making anything private. It's just for you and for other users of your code.

**Comment by ShatReal** [comment]
Yeah I'm trying to make sure I stick to a style convention. But that just ends up with nearly every variable with an underscore at the front lol
